highfidelity/hifi
# Adding Group Teleport to Default Script

To Test Group Teleport:

1) Get a few avatars in a domain. Make sure you have edit permission.
2) Press both thumbpads down on vive/press space bar to activate group teleport. You should see a plain rug under your feet.
3) Invite the other avatars to step on the rug.
4) Then you may teleport as you normally would with the laser beam. After you teleported, you shall see that other avatars will also teleport with you.

To Test Customization:

1) Click Settings > Customize Group Teleport
2) That will bring up a prompt you can enter a URL for a model. The model is expected to be in the dimension of  x: 2.0, y: 0.0067, z: 2.0.
3) For testing, you can enter this URL for this other model I've prepared: https://s3-us-west-1.amazonaws.com/hifi-content/faye/earth.fbx
4) Click OK.
5) Now try group teleport again, you shall see that the rug is now circle and has earth pattern on.

Lmk if there's anything unclear. Thanks!
